app.go

```diff
@@ -396,6 +402,9 @@ func New(config ...Config) *App {
 	if app.config.JSONEncoder == nil {
 		app.config.JSONEncoder = json.Marshal
 	}
+	if app.config.Network == "" {
+		app.config.Network = "tcp4"
+	}
```

Comment:
Wouldn't it be better if you keep this strings (`tcp`, `tcp4`, `tcp6`) as constant and use the constant everywhere?
Also make those constants exported, so that when I use the network option, i can use it like `Network: fiber.NetworkTCP4` instead of `Network: "tcp4"`. And typing hardcoded strings can be misspelled.

Reply:
I like the idea, but now that the PR has been approved, would you send in another to fix it pls @hsblhsn?

Reply:
@suntong This PR is approved but not merged yet, I think @kiyonlin can add this little change in this PR. Or I have to wait until this PR merges. I don't want to open a PR with duplicated code from another PR with just a little bit of change. Whatever, I will try to open a PR if @kiyonlin don't add it in this PR.
Thank you!

Reply:
gotya

Reply:
> Wouldn't it be better if you keep this strings (`tcp`, `tcp4`, `tcp6`) as constant and use the constant everywhere?
> 
> Also make those constants exported, so that when I use the network option, i can use it like `Network: fiber.NetworkTCP4` instead of `Network: "tcp4"`. And typing hardcoded strings can be misspelled.

Good idea! Will improve this.